apps/files_external/controller/storagescontroller.php

```
@@ -242,10 +242,44 @@ protected function updateStorageStatus(StorageConfig &$storage) {
 				$this->l10n->t('Insufficient data: %s', [$e->getMessage()])
 			);
 		} catch (StorageNotAvailableException $e) {
-			$storage->setStatus(
-				\OC_Mount_Config::STATUS_ERROR,
-				$e->getMessage()
-			);
+			switch ($e->getCode()) {
+				case 1:
```

Just thought it might be good to have these constants directly in the `StorageNotAvailableException`. This way the apps/storages that want to throw the exception can directly use these constants while throwing.